output/source-info.ts

```diff
@@ -137,6 +137,13 @@ const D2Sources: {
     ],
     searchString: [],
   },
+  brave: {
+    itemHashes: [],
+    sourceHashes: [
+      2952071500, // Source: Into the Light
```

Comment:
> This seems to have pulled in only 7 of the 12 Brave weapons. Not sure what happened here.
this repo helps categorize things. the result here ^ is "things that have this source hash, are `brave`"